packages/graphql/tests/integration/issues/6005-filters.int.test.ts

```diff
@@ -0,0 +1,281 @@
+/*
```

Comment:
Per convention, this test should simply be called `6005.int.test.ts` and the name should only contain the url to the issue with the full context

Reply:
This is because this tests is related only to filters, the test for 6005 should go on for the 6.x branch with the count fix projection

Reply:
Still, that test should eventually be merged into the same file no?

Maybe a TODO or a comment is better suited than having a renamed file?